pubsub/pubsub_goredis.go

```diff
@@ -69,6 +70,20 @@ func (ps *GoRedisPubSub) Start() error {
 		options.Username = username
 		options.Password = pw
 		options.DB = ps.Config.GetRedisDatabase()
+		useTLS := ps.Config.GetUseTLS()
+		tlsInsecure := ps.Config.GetUseTLSInsecure()
+		if useTLS {
+			tlsConfig := &tls.Config{
+				MinVersion: tls.VersionTLS12,
+			}
+
+			if tlsInsecure {
+				tlsConfig.InsecureSkipVerify = true
+			}
+
+			options.TLSConfig = tlsConfig
+		}
+
```

Comment:
I notice that ps.Config.Get* functions return two things: a value and an error.

What do you think of:

```suggestion
		useTLS, err := ps.Config.GetUseTLS()
		if err != nil {
			return err
		}
		if useTLS {
			tlsInsecure, err := ps.Config.GetUseTLSInsecure()
			if err != nil {
				return err
			}
			options.TLSConfig = &tls.Config{
				MinVersion: tls.VersionTLS12,
				InsecureSkipVerify: tlsInsecure,
			}
		}
```

Reply:
I thought we have fixed the config interface for all getters to not return an error?

Reply:
Ah. Yes, on `main`. I was looking at this code path in `v2.7.0` which still have the error returns.

Then how about? … 

```go
		if ps.Config.GetUseTLS() {
			options.TLSConfig = &tls.Config{
				MinVersion: tls.VersionTLS12,
				InsecureSkipVerify: ps.Config.GetUseTLSInsecure(),
			}
		}
```